spec/cdn_source_spec.rb

```diff
@@ -71,8 +71,48 @@ def all_local_files
       it 'raises if unexpected HTTP error' do
         REST.expects(:get).returns(REST::Response.new(500))
         should.raise Informative do
-          @source.versions('Unknown_Pod')
-        end.message.should.match /CDN: .* Relative path couldn\'t be downloaded: .* Response: 500/
+          @source.specification('BeaconKit', '1.0.0')
+        end.message.
+          should.== "CDN: #{@source.name} Relative path couldn\'t be downloaded: Specs/2/0/9/BeaconKit/1.0.0/BeaconKit.podspec.json Response: 500"
```

Comment:
nit: remove trailing `.` after `should`, a few other instances below as well

Reply:
Funnily enough, the syntax for matching raised exceptions doesn't allow that. Very strange but you're welcome to try it yourself 😂

I changed the one place where it isn't a raised exception.